examples/animatediff/args_train.py

```diff
@@ -170,12 +170,22 @@ def parse_args():
     parser.add_argument(
         "--motion_module_path", default="", type=str, help="path to pretrained motion mdule. Load it if not empty"
     )
+    parser.add_argument("--train_data_type", default="video_file", type=str, help="type of data for training")
```

Comment:
add `choices=[]` to clarify all the possible train_data_type

Reply:
fixed